src/main/java/techcourse/myblog/domain/service/ArticleService.java

```diff
@@ -0,0 +1,60 @@
+package techcourse.myblog.domain.service;
```

Comment:
계층형 아키텍처에서 응용 영역과 도메인 영역은 동일한 수준의 영역입니다.

```suggestion
package techcourse.myblog.service;
```

Reply:
수정했습니다!